docs/install-manual.md

```diff
@@ -0,0 +1,144 @@
+This section tells you how to create a Docker Swarm on your network to use only for debugging and testing purposes. You can also use this type of installation if you are developing custom applications for Docker Swarm or contributing to it.  
+
+> **Caution**: Only use this set up if your network environment is secured by a firewall or other measures.
+
+## Prerequisites 
+
+You install Docker Swarm on a single system which is known as your Docker Swarm manager. You create the cluster, or swarm, on one or more additional nodes on your network. Each node in your swarm must:
+
+* be accessible by the swarm manager across your network
+* the same TCP port open to listen for the swarm manager  (2375)
+
+You can run Docker Swarm on Linux 64-bit architectures. You can also install and run it on 64-bit Windows and Max OSX but these architectures are *not* regularly tested for compatibility in the BETA phase.
+
+Take a moment and identify the systems on your network that you intend to use. Ensure each node meets the requirements listed above.
+
+
+
+## Pull the swarm image and create a cluster.
+
+The easiest way to get started with Swarm is to use the
+[official Docker image](https://registry.hub.docker.com/_/swarm/).
+
+1. Pull the swarm image.
+
+		$ docker pull swarm
+
+1. Create a Swarm cluster using the `docker` command.
+
+    ```bash
+    $ docker run --rm swarm create
+    6856663cdefdec325839a4b7e1de38e8 # 
+    ```
+
+    The `create` command returns a unique cluster ID (`cluster_id`). You'll need
+    this ID when starting the Docker Swarm agent on a node.
+
+##  Create swarm nodes
+
+Each Swarm node will run a Swarm node agent. The agent registers the referenced
+Docker daemon, monitors it, and updates the discovery backend with the node's status.
+
+This example uses the Docker Hub based `token` discovery service. Log into **each node** and do the following.
+
+1. Start the Docker daemon with the `-H` flag. This ensures that the Docker remote API on *Swarm Agents* is available over TCP for the *Swarm Manager*.
+
+    		$ docker -H tcp://0.0.0.0:2375 -d
+ 
+
+2. Register the Swarm agents to the discovery service. The node's IP must be accessible from the Swarm Manager. Use the following command and replace with the proper `node_ip` and `cluster_id` to start an agent:
+
+		```bash
+		docker run -d swarm join --addr=<node_ip:2375> token://<cluster_id>
+		```
+
+  For example:
+
+		$ docker run -d swarm join --addr=172.31.40.100:2375 token://6856663cdefdec325839a4b7e1de38e8
+
+3. Start the Swarm manager on any machine or your laptop. 
+
+	The following command illustrates how to do this:
+
+	```bash
+	docker run -d -p <swarm_port>:2375 swarm manage token://<cluster_id>
+	```
+
+4. Once the manager is running, check your configuration by running `docker info` as follows:
+
+	```bash
+	docker -H tcp://<manager_ip:manager_port> info
+	```
+
+	For example, if you run the manager locally on your machine:
+
+      $ docker -H tcp://0.0.0.0:2375 info
```

Comment:
(Viewing on GitHub) This this block needs to be indented with one more space; it isn't rendered as code-block